docs/guides/getting-started.md

```diff
@@ -20,11 +20,11 @@ page_title: Getting Started
     sudo apt-get update && sudo apt-get install terraform
     ```
 
-- Create an account with any supported cloud vendor and expose its [authentication credentials via environment variables][authentication]
+- Create an account with any supported cloud vendor and expose its [authentication credentials via environment variables][auth]
 
-[authentication]: https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/authentication
+[auth]: https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/authentication
```

Comment:
BTW I think the nav would be more logical with Getting Started on top, since it covers installation and then it links to the auth section.